app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/util/CipherViewExtensionsTest.kt

```diff
@@ -15,13 +15,14 @@ import org.junit.jupiter.api.Test
 
 class CipherViewExtensionsTest {
 
+    private val id = "mockId-1"
+    private val username = "Bitwarden"
+    private val password = "password"
+    private val totpCode = "mockTotp-1"
+    private val uri = "www.test.com"
+
     @Test
-    fun `toOverflowActions should return all actions for a login cipher`() {
-        val id = "mockId-1"
-        val username = "Bitwarden"
-        val password = "password"
-        val totpCode = "mockTotp-1"
-        val uri = "www.test.com"
+    fun `toOverflowActions should return all actions for a login cipher when a user has premium`() {
```

Comment:
Just want to callout this is the test class where I am testing the logic change, the rest of test edits are just including the necessary value in the mock data to compile and run the existing tests which are not checking this condition :)